content/en/tracing/llm_observability/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: LLM Observability
+kind: documentation
+---
+
+<div class="alert alert-info">LLM Observability is in public beta.</a></div>
+
+{{< img src="tracing/llm_observability/llm-observability-overview.png" alt="An LLM Observability trace displaying each span of a request" style="width:100%;" >}}
+
+With LLM Observability, you can monitor the quality, security, and performance of your LLM applications. Each request fulfilled by your application is represented as a trace on the [LLM Observability traces list page][2] in Datadog.
+
+A trace can represent:
+- An individual LLM inference, including tokens, error rates, latencies
+- A predetermined LLM *workflow*, a grouping of LLM calls and their contextual operations such as tool calls or preprocessing steps
+- A dynamic LLM workflow executed by an LLM agent
+
+Each trace contains spans representing each choice made by an agent or each step of a given workflow. A given trace can also include input and output, execution duration, privacy issues, errors, and more.
+
+## Getting started
+
+{{< whatsnext desc="Get started with LLM Observability:" >}}
```

Comment:
@barieom, we don't want to route users to reference docs that don't have a linear order -- a common complaint from users is that we sent them to reference material without telling them what to actually do. So we want to route them to guide docs that will walk them through the process step by step, which both of the provided links do. But I'll verify that the SDK and API docs are linked from both of those guide docs.

Reply:
@jhgilbert gotcha! in that case, it's fine if we don't route them out to the docs directly, but let's explicitly mention that we offer these 2 ways of instrumenting the app here because as a new user, i'd want to know that up front